src/components/CortexPage/CortexPage.tsx

```diff
@@ -53,35 +55,51 @@ export const CortexPage = ({
   const canEditSettings =
     permissions && permissions?.permissions.includes(Permission.EDIT_SETTINGS);
 
-  if (loadingPermissions || loadingHelpPage) {
+  const {
+    value: expiration,
+    loading: loadingExpiration,
+  } = useAsync(async () => {
+    return await cortexApi.getExpiration();
+  }, []);
+
+  if (loadingPermissions || loadingHelpPage || loadingExpiration || isNil(expiration)) {
     return <Progress />;
   }
 
   return (
-    <CortexLayout title={title} subtitle={subtitle}>
-      <CortexLayout.Route path="scorecards" title="Scorecards">
-        <ScorecardsPage />
-      </CortexLayout.Route>
-      <CortexLayout.Route path="reports" title="Reports">
-        <ReportsPage />
-      </CortexLayout.Route>
-      <CortexLayout.Route path="initiatives" title="Initiatives">
-        <InitiativesPage />
-      </CortexLayout.Route>
-      {/*
-      Show the settings page if there is an error (will occur when email header authorization returns an error, which is a valid use case)
-      or if the user has the EDIT_SETTINGS permission
-       */}
-      {!hideSettings && (!isNil(permissionsError) || canEditSettings) && (
-        <CortexLayout.Route path="settings" title="Settings">
-          <SettingsPage />
-        </CortexLayout.Route>
-      )}
-      {!isNil(helpPage) && (
-        <CortexLayout.Route path="help" title="Help">
-          <HelpPage helpPage={helpPage} />
-        </CortexLayout.Route>
+    <>
+      {shouldShowExpirationBanner(expiration) && (
+        <ExpirationBanner {...expiration} />
       )}
-    </CortexLayout>
+      {
+        isBeforeShutdownDate(expiration) && (
+          <CortexLayout title={title} subtitle={subtitle}>
```

Comment:
Can we put the banner below the Cortex title? And still show the Cortex title even if it's expired

Reply:
Backstage theme we use has fixed grid areas. 
```
grid-template-areas: 'pageHeader pageHeader pageHeader' 'pageSubheader pageSubheader pageSubheader' 'pageNav pageContent pageSidebar';
```

To change that we would need to override it and put something like `pageBanner` between `pageHeader` (which rendering main Cortex Banner) and `pageSubheader` (which renders tabs). Something like:
```
grid-template-areas: 'pageHeader pageHeader pageHeader' 'pageBanner pageBanner pageBanner' 'pageSubheader pageSubheader pageSubheader' 'pageNav pageContent pageSidebar';
```

To do this we would need to create our custom MUI theme. We spent some time to do this with Igor but to no avail. Tbh, looks like a lot of stuff to change. My proposal would be to keep the ExpirationBanner on top but still show Cortex header when no access:

![image](https://github.com/cortexapps/backstage-plugin/assets/138560262/e9b781f0-2a6e-4ec7-b687-6f98220118a8)

![image](https://github.com/cortexapps/backstage-plugin/assets/138560262/bf0e58e8-9856-44c8-8b2a-d9c16f8c3024)